example-deck.gl.md

```diff
@@ -0,0 +1,400 @@
+</style>
+<div id="container"></div>
+
+<script src="https://unpkg.com/deck.gl@latest/dist.min.js"></script>
+```
+
+Now we can start to write our JavaScript code.
+
+Deck.gl will combine points in the same hexagon to an array and let the user define how the elevation and color of this hexagon is calculated.
+We choose to use the average of all points in a hexagon for this.
+So we need a small helper function:
+
+```javascript
+// Helper function to calculate the average value
+function average(numbers) {
+    let total = 0;
+    for (let i = 0; i < numbers.length; i += 1) {
+        total += numbers[i];
+    }
+    return total / numbers.length;
+}
+```
+
+Now let's define the visualization layer we have seen above:
+
+```javascript
+new deck.HexagonLayer({
+    extruded: true,
+    radius: 5000,
+    data: API_URL,
+    dataTransform: d => d.features,
+    elevationScale: 100,
+    getColorValue: points => average(points.map(d => d.properties.carbonmonoxide)),
+    getElevationValue: points => average(points.map(d => d.properties.carbonmonoxide)),
+    getPosition: d => d.geometry.coordinates,
+}),
+```
+
+We can give the API URL directly to the *HexagonLayer* and can extract the array of the different points with *dataTransform*.
+With *getPosition* we calculate the coordinates for every point and with *getElevationValue* and *getColorValue* we can directly map the array of points in a hexagon to its elevation and color.
+For a closer look, see [API Reference HexagonLayer](https://deck.gl/#/documentation/deckgl-api-reference/layers/hexagon-layer).
+
+At last we create the *DeckGL* scripting object:
+
+```javascript
+new deck.DeckGL({
+    container: 'container',
+    mapboxApiAccessToken: '<your mapbox token>',
+    mapStyle: "mapbox://styles/mapbox/dark-v9",
+    longitude: 10,
+    latitude: 50,
+    zoom: 5,
+    pitch: 50,
+    layers: [
+        …
+    ]
+});
+```
+
+Most parameters control the initial view and the underlying map.
+*container* is the HTML-element we defined earlier and the visualization layers are in *layers*.
+For details about the different parameter, take a look at [API Reference for DeckGL](https://deck.gl/#/documentation/deckgl-api-reference/scripting-interface/deckgl).
+
+## Bringing It All Together
+
+Finally, let’s take all the parts and bring them together.
+
+For this, we can just copy most of the code we already have.
+
+### HTML
+
+```html
+<script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.js'></script>
+<link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css' rel='stylesheet' />
+<script src="https://unpkg.com/deck.gl@latest/dist.min.js"></script>
+
+<style type="text/css">
+      #container {width: 100%; height: 500px;}
+</style>
+<div id="container"></div>
+```
+
+### JavaScript
+
+```javascript
+// A polygon describing the vague form of Germany
+const GERMANY = 'polygon=9.921906365609232,54.983104153048025,' +
+    '9.9395797054529,54.596641954153256,' +
```

Comment:
Can we get/provide this as a JS library? 😸

Reply:
I am not aware of one, but we should take a look.

Reply:
At a first glance, I did not find a JavaScript library for that, but I did not look very hard. Since this is probably temporary (as far as I know @benszau is creating a patch for that), I would suggest leaving it like that.